src/atoms/feedCardItem/transactionTypes/liquidityTransactionCard.tsx

```diff
@@ -116,9 +146,9 @@ export const LiquidityTransactionCard = (
         </div>
         <XYPartyContent>
           <HintsAndHovers
-            id={token1AmountUsd.toString()}
+            id={uuidv4()}
             hint={dollarPopover}
-            icon={getFormattedText(token1AmountUsd, 'S-Regular')}
+            icon={getFormattedText(getTotal(), 'S-Regular')}
```

Comment:
@shan8851 it can be expensive it run a function into the component , do console.log into the getTotal to be able to understand how many times it will call it

Reply:
@goshatravin  so it only fires once initially, but does refire when we hover over component. Timur did suggest we can calculate this on the backend if needed, so if we think this is going to be an issue performance wise then we can maybe look at doing that.